modules/managed-cluster-add-notification-contacts.adoc

```diff
@@ -27,3 +27,11 @@ By default, only the cluster owner receives cluster notification emails. You can
 
 .Verification steps
 * The "Notification contact added successfully" message displays.
+
+.Troubleshooting
+
+Error: `Account with ID <username> denied access to perform create on NotificationContact`::
+This error occurs when you are logged in to the cluster using an account that does not have the cluster owner or cluster editor roles. Log in to a different account that has these roles and try again.
+
+Error: `Could not find any account identified by <email-address>`::
```

Comment:
@laubai Hi, The notification contact could be added with RedHat username  or email address. Hence the error message could be "**Could not find any account identified by <username |email-address>**". Kindly check.